datahub-graphql-core/src/main/resources/app.graphql

```diff
@@ -379,6 +379,16 @@ type FeatureFlagsConfig {
   Right now this only affects ability to edit user profile image URL but can be extended.
   """
   readOnlyModeEnabled: Boolean!
+
+  """
+  Whether search filters V2 should be shown or the default filter side-panel
+  """
+  showSearchFiltersV2: Boolean!
+
+  """
+  Whether browse V2 sidebar should be shown
+  """
+  showBrowseV2: Boolean!
```

Comment:
This is only applicable if showSearchFiltersV2 is set to true right? Or no?

Reply:
they're independent right now, but we don't recommend anyone turn browse on without search on

Reply:
as a followup we could build it in code so that if search isn't on, browse is off regardless of what the env variable is